src/main/dialogs/basic/dialog.js

```diff
@@ -92,7 +99,8 @@ fluid.defaults("gpii.app.dialog", {
         closable: false,
```

Comment:
Choose a different name which better conveys the idea of this property.

Reply:
I suppose this to be done in a different PR